Tribler/Main/tribler_main.py

```diff
@@ -943,7 +943,6 @@ def OnExit(self):
             self._logger.info("ONEXIT Session is shutdown")
 
         self.utility.session.notifier.notify(NTFY_CLOSE_TICK, NTFY_INSERT, None, 'Deleting instances')
-        wx.Yield()
```

Comment:
Yeah but it does not fail the tests themselves (it trips the watchdog however). It only happens on "big" tests that are running Tribler for a while (such as channel search).